pyNN/common/populations.py

```diff
@@ -828,6 +832,7 @@ def __init__(self, parent, selector, label=None):
         self.local_cells = self.all_cells[self._mask_local]
         self.first_id = numpy.min(self.all_cells)  # only works if we assume all_cells is sorted, otherwise could use min()
         self.last_id = numpy.max(self.all_cells)
+        self.annotations = {}
```

Comment:
ok, fair enough. Probably then we should move the `annotate()` method to `BasePopulation`